Add how to use puffin http (#105)
EmbarkStudios/puffin: puffin_http/README.md

A HTTP server/client for communicating [`puffin`](https://github.com/EmbarkStudios/puffin) profiling events.

You can view them using [`puffin_viewer`](https://github.com/EmbarkStudios/puffin/tree/main/puffin_viewer).

## How to use

> In addition to puffin stuff, you need to create a `puffin_http` Server into the application you want to profile.

Author: You think it's better to simplify to this :
> Add a `puffin_http` `Server` to the profiled application.

When the server is started, [`puffin_viewer`](https://crates.io/crates/puffin_viewer) application can connect to it and display profiling informations.

``` rust
fn main() {
    let server_addr = format!("0.0.0.0:{}", puffin_http::DEFAULT_PORT);
    puffin_http::Server::new(&server_addr).unwrap();
}
```

> You can look into examples/server.rs for more complete example.

Reviewer: ```suggestion
You can checkout the examples/server.rs for more complete examples.
```

Author: Perhaps :
You can checkout the examples/server.rs for a more complete example.
